plugins/destination/mysql/cloud-config-ui/src/App.tsx

```diff
@@ -19,10 +19,19 @@ const useCloudAppMock =
   window.self === window.top;
 const DevWrapper = useCloudAppMock ? CloudAppMock : Fragment;
 // eslint-disable-next-line unicorn/prefer-module
-const devWrapperProps: any = useCloudAppMock ? require('./.env.json') : undefined;
+const { plugin, ...devWrapperProps }: any = useCloudAppMock ? require('./.env.json') : {};
+
+const pluginProps = useCloudAppMock
+  ? plugin
+  : {
+      team: process.env.REACT_APP_PLUGIN_TEAM,
```

Comment:
I think you'll have to add these env vars to the .github/workflows/dest_mysql.yml

Reply:
Nice, forgot about that, thanks!